src/main/java/com/wdtinc/mapbox_vector_tile/adapt/jts/MvtReader.java

```diff
@@ -230,7 +235,11 @@ private static Geometry readPoints(GeometryFactory geomFactory, List<Integer> ge
             nextCoord.setOrdinate(1, cursor.y);
         }
 
-        return coordSeq.size() == 1 ? geomFactory.createPoint(coordSeq) : geomFactory.createMultiPoint(coordSeq);
+        if(coordSeq.size() == 1) {
```

Comment:
Ternary operator should be compatible with Android?

Reply:
ternary operators are fine with Android - it was above the 100 character Google check limit.